src/scattdata_header.F90

```diff
@@ -410,13 +410,16 @@ subroutine scattdatatabular_init(this, mult, coeffs)
                  this % fmu(gin) % data(:, gout) / norm
           end if
 
-          ! Now create CDF from fmu with trapezoidal rule
+          ! Now create CDF from fmu with the analytical integral
           this % dist(gin) % data(1, gout) = ZERO
           do imu = 2, order
-            this % dist(gin) % data(imu, gout) = &
-                 this % dist(gin) % data(imu - 1, gout) + &
-                 HALF * this % dmu * (this % fmu(gin) % data(imu - 1, gout) + &
-                                      this % fmu(gin) % data(imu, gout))
+            p0 = this % fmu(gin) % data(imu - 1, gout)
+            mu0 = this % mu(imu - 1)
+            mu1 = this % mu(imu)
+            m = (this % fmu(gin) % data(imu, gout) - p0) / (mu1 - mu0)
+            this % dist(gin) % data(imu, gout) = HALF * m * mu1 * mu1 + &
+                 (p0 - m * mu0) * mu1 + &
+                 (HALF * m * mu0 * mu0 - p0 * mu0)
```

Comment:
Same question here -- to me analytical integral is same thing as trapezoidal rule (which exactly integrates a linear function)